content/blog/2025-04-09-introducing-bloom-filters.md

```diff
@@ -148,7 +148,7 @@ The other bloom filter commands are O(1) time complexity: `BF.CARD`, `BF.INFO`,
 valkey-bloom offers an efficient solution for high-volume membership testing through bloom filters, providing significant memory usage savings compared to traditional data types.
 This enhances Valkey's capability to handle various workloads including large-scale advertisement / event deduplication, fraud detection, and reducing disk / backend lookups more efficiently.
 
-To learn more about [valkey-bloom](https://github.com/valkey-io/valkey-bloom/), you can read about the data type [here](https://valkey.io/topics/bloomfilters/) and follow the [quick start guide](https://github.com/valkey-io/valkey-bloom/blob/1.0.0/QUICK_START.md) to try it yourself.
+To learn more about [valkey-bloom](https://github.com/valkey-io/valkey-bloom/), you can read about the [Bloom Filters data type](https://valkey.io/topics/bloomfilters/) and follow the [quick start guide](https://github.com/valkey-io/valkey-bloom/blob/1.0.0/QUICK_START.md) to try it yourself.
```

Comment:
it's worth thinking about how to correctly express name links, I've picked things that should be seen as definite improvements over what was here, but I'm not claiming they're perfect...

Reply:
Yeah, these make sense to me. We can keep an eye for it moving forward.